docs/security/seccomp.md

```diff
@@ -0,0 +1,64 @@
+<!-- [metadata]>
++++
+title = "Seccomp security profiles for Docker"
+description = "Enabling seccomp in Docker"
+keywords = ["seccomp, security, docker, documentation"]
++++
+<![end-metadata]-->
+
+Seccomp security profiles for Docker
+------------------------------------
+
+The seccomp() system call operates on the Secure Computing (seccomp)
+state of the calling process.
+
+This operation is available only if the kernel is configured
```

Comment:
Hi Jess, don't worry, I'm updating it. I just wanted to check.  In looking at the history are you attaching the flag `--securty-opt` to `run` or the `daemon` start? I need to update the command.